src/modules/GroupChannel/components/MessageInputWrapper/index.tsx

```diff
@@ -1,81 +1,113 @@
 import './index.scss';
-import React, { useState, useContext, useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
+import type { User } from '@sendbird/chat';
+import type { GroupChannel } from '@sendbird/chat/groupChannel';
+import type {
+  FileMessage,
+  FileMessageCreateParams,
+  MultipleFilesMessage,
+  MultipleFilesMessageCreateParams,
+  UserMessage,
+  UserMessageCreateParams,
+} from '@sendbird/chat/message';
 
-import * as utils from '../../context/utils';
-
-import type { Nullable } from '../../../../types';
-import MessageInput from '../../../../ui/MessageInput';
-import QuoteMessageInput from '../../../../ui/QuoteMessageInput';
-import { LocalizationContext } from '../../../../lib/LocalizationContext';
-import { useGroupChannelContext } from '../../context/GroupChannelProvider';
+import {
+  isOperator as isChannelOperator,
+  isDisabledBecauseFrozen,
+  isDisabledBecauseMuted,
+} from '../../context/utils';
 import useSendbirdStateContext from '../../../../hooks/useSendbirdStateContext';
+import { useLocalization } from '../../../../lib/LocalizationContext';
 import SuggestedMentionList from '../SuggestedMentionList';
-import { MessageInputKeys } from '../../../../ui/MessageInput/const';
-import VoiceMessageInputWrapper from './VoiceMessageInputWrapper';
 import { useDirtyGetMentions } from '../../../Message/hooks/useDirtyGetMentions';
+import { SendableMessageType } from '../../../../utils';
+import QuoteMessageInput from '../../../../ui/QuoteMessageInput';
+import VoiceMessageInputWrapper from './VoiceMessageInputWrapper';
+import MessageInput from '../../../../ui/MessageInput';
 import { useMediaQueryContext } from '../../../../lib/MediaQueryContext';
+import { MessageInputKeys } from '../../../../ui/MessageInput/const';
 import { useHandleUploadFiles } from './useHandleUploadFiles';
 
-export type MessageInputWrapperProps = {
```

Comment:
It seems like there might be issues with the existing customers' code if the props (interface) of this component are changed, as we are already exporting this component.
The current code should be written to be the `GroupChannelInputWrapperView`.

```tsx
const ChannelInputWrapper = ({ value, disabled }) => {
  const { currentChannel, isMultipleFilesMessageEnabled } = useChannelContext();
  return <GroupChannelInputWrapperView currentChannel={currentChannel} {...} />
}
```